grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/ArmeriaClientCall.java

```diff
@@ -424,7 +425,7 @@ public void onNext(DeframedMessage message) {
                           new Metadata());
                 } else {
                     GrpcWebTrailers.set(ctx, trailers);
-                    GrpcStatus.reportStatus(trailers, this);
+                    DefaultGrpcExceptionHandlerFunction.reportStatus(trailers, this);
```

Comment:
`reportStatus` is unrelated to `GrpcExceptionHandlerFunction`. Should we leave it in `GrpcStatus` and only move code handling exceptions to `DefaultGrpcExceptionHandlerFunction`?

Reply:
Thanks for the recomendation. I also think we should separate `GrpcStatus` and `DefaultGrpcExceptionHandlerFunction`.

Reply:
Applied feedback. Thanks